src/huggingface_hub/keras_mixin.py

```diff
@@ -59,9 +173,13 @@ def save_pretrained_keras(
         with open(path, "w") as f:
             json.dump(config, f)
 
+    hyperparameters = _extract_hyperparameters_from_keras(model)
```

Comment:
Why do we do this within `save_pretrained_keras` and return to `push_to_hub_keras`? Can this be done directly in `push_to_hub_keras`? I feel that makes more sense and the model is also available there

Reply:
Sure I'll do them in `push_to_hub_keras()` instead.